spec/features/admin/orders_spec.rb

```diff
@@ -145,6 +145,17 @@ def new_order_with_distribution(distributor, order_cycle)
     expect(page).to have_selector 'p', text: "Order cycle: #{@order.order_cycle.name}"
   end
 
+  scenario "can't print invoices when invocies are disabled" do
```

Comment:
Could you do this in a view spec?
(It feels like Pau is speaking through me. :grin:)

Reply:
Now I see you already linked this to the view specs pr.

Reply:
I did it in a view spec, I thought of deleting this but then left it. should I delete this spec?

this same verification in a view spec:
https://github.com/openfoodfoundation/openfoodnetwork/pull/3962/files#diff-271f5821d793560402267a225b586263R31

Reply:
Yes, I think that this is completely covered by the view spec.

Reply:
ok, deleted now.